chart/kubeapps/values.schema.json

```diff
@@ -0,0 +1,84 @@
+{
+  "$schema": "http://json-schema.org/schema#",
+  "type": "object",
+  "properties": {
+    "frontend": {
+      "type": "object",
+      "title": "Frontend configuration",
+      "form": true,
+      "properties": {
+        "replicaCount": {
```

Comment:
I would expose the replica count of the other services too: `tiller-proxy`, `chartsvc`, `dashboard` (not the apprepository-controller which should be 1).